src/components/DataTable/tools/filter.js

```diff
@@ -1,5 +1,17 @@
 import { getCellId } from './cells';
 
+/**
+ * Default implemention of how we filter rows internally. The idea behind this
+ * implementation is to use the given list of row ids and headers to get the
+ * individual cell values for a row. Then, we go through each cell value and see
+ * if any of them includes the given inputValue.
+ *
+ * @param {Array[string]} rowIds array of all the row ids in the table
```

Comment:
Great to have this as something like `Array<string>` or `string[]` so that more JSDoc-based tools would understand it.

Reply:
Makes sense. Thanks @asudoh!